server/service/integration_mdm_test.go

```diff
@@ -12753,11 +12758,14 @@ INSERT INTO host_mdm_apple_declarations (
 			TeamID:          ptr.Uint(0),
 			Identifier:      "com.example",
 			Name:            "Example",
-			DeclarationType: fleet.MDMAppleDeclarativeConfiguration,
-			Declaration:     json.RawMessage(`{"foo": "bar"}`),
-			MD5Checksum:     "csum123",
-			CreatedAt:       then,
-			UploadedAt:      then,
+			Category:        fleet.MDMAppleDeclarativeConfiguration,
+			RawJSON: json.RawMessage(`{
+				"Type": "com.apple.configuration.declaration-items.test",
+				"Payload": {"foo":"bar"},
+				"Identifier": "com.example"
```

Comment:
Should we add the IT admin's `ServerToken` to the raw and explicitly test that it is disregarded? I think that would make it clearer to future contributors that it is the expected behavior. We can circle back to add in tests if you'd like to get this merged now.